tests/utils/dockercli/volumelifecycle.go

```diff
@@ -53,8 +65,45 @@ func KillDocker(ip string) ([]byte, error) {
 	return out, err
 }
 
+// KillVDVSPlugin - kill vDVS plugin. It is restarted automatically
+func KillVDVSPlugin(ip string) ([]byte, error) {
+	log.Printf("Killing vDVS plugin on VM [%s]\n", ip)
+	out, err := ssh.InvokeCommand(ip, dockercli.GetVDVSPlugin)
+	if err != nil {
+		return out, err
+	}
+
+	pluginID := strings.Fields(misc.FormatOutput(out))[0]
+	out, err = ssh.InvokeCommand(ip, dockercli.KillVDVSPlugin+pluginID)
+
+	time.Sleep(2 * time.Second)
+
+	out, err = ssh.InvokeCommand(ip, dockercli.GetVDVSPlugin)
```

Comment:
exposed 3 separate utils: get plugin, kill plugin and start plugin. Wrapped it into one util to be called from test. rest utils can be used in future as per required.